plugins/gradle/src/main/groovy/com/newrelic/agent/android/ClassTransformWrapperTask.groovy

```diff
@@ -59,8 +60,16 @@ abstract class ClassTransformWrapperTask extends DefaultTask {
                                 def jarEntry = new JarEntry(relativePath.replace(File.separatorChar, '/' as char))
                                 try {
                                     jarOutputStream.putNextEntry(jarEntry)
-                                    transformer.processClassBytes(classFile, fileInputStream).withCloseable {
-                                        jarOutputStream << it
+                                    try {
+                                        transformer.processClassBytes(classFile, fileInputStream).withCloseable {
+                                            jarOutputStream << it
+                                        }
+                                    } catch (RuntimeException re) {
+                                        logger.warn("[ClassTransformTask] encountered a RuntimeException: " + re.getMessage() + ", output the original jar file")
```

Comment:
How about "_instrumentation is disabled for <class file>_"? Also, don;t seend to say both "_encountered a RuntimeException_" and "_re.getMessage()_". The exception message is sufficient.

Reply:
Does this sound better/
`logger.warn("[ClassTransformTask] Instrumentation is disabled for " + classFile.name + " with exception: "+ re.getMessage())`